example_scripts/download_vhi_parallel.py

```diff
@@ -84,9 +93,12 @@ def chunks(l, n):
         yield l[i:i+n]
 
 
-def main(pool):
+def main(pool, subset=False):
```

Comment:
added this subset so that Gabi you can test the code (literally just takes a small number of the files...)

Reply:
@gabrieltseng should we remove ?

Reply:
Maybe we could replace this with an argument that accepts an `int` (or `None`) instead of a `bool`, and allows users to decide how many examples to limit downloads to.